stop-mycroft.sh

```diff
@@ -42,7 +43,7 @@ function help() {
 }
 
 function process-running() {
-    if [[ $( ps aux | grep "[p]ython .*${1}/main.py" ) ]] ; then
+    if [[ $( ps aux | grep "[p]ython[2|3]\? .*${1}/main.py" ) ]] ; then
```

Comment:
The regex should be `[23]` (even though it won't fail by allowing it to match `|`)